qiskit/providers/aer/noise/device/models.py

```diff
@@ -249,11 +249,11 @@ def _excited_population(freq, temperature):
     if freq != inf and temperature != 0:
         # Compute the excited state population from qubit
         # frequency and temperature
-        # Boltzman constant  kB = 6.62607015e-34 (eV/K)
-        # Planck constant h =  6.62607015e-34 (eV.s)
+        # Boltzman constant  kB = 8.617333262-5 (eV/K)
+        # Planck constant h = 4.135667696e-15 (eV.s)
```

Comment:
didn't want to change the units to J.S?

Reply:
That would have also fixed it, but this way they are in compatible units for computing h/kb